.github/actions/reopenIssueWithComment/action.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Reopen issue with comment'
+description: 'Reopens a GitHub issue and leaves a comment.'
+inputs:
+  GITHUB_TOKEN:
+    description: Auth token for Expensify.cash Github; necessary for accessing Octokit.
```

Comment:
I like this description better than the one you used in `checkDeployBlockers/action.yml`